lib/api_delegator.rb

```diff
@@ -1,22 +1,21 @@
-class Api
-  def self.request(system, endpoint, args = {})
-    Api
+module ApiDelegator
```

Comment:
[praise] I love the change from `Api` to `ApiDelegator`

Reply:
Well, since it doesn't behave like a class, we're not going to need instance methods, and it's not being inherited by anything, it made sense to me to just collect a couple methods up in a module.  I don't feel terribly strongly about it, though, so if you would prefer a class I can change that.

Reply:
Nah, that all sounds like a good reason to make it a module.